registrations/api.py

```diff
@@ -220,8 +221,131 @@ def send_invitation(self, request, pk=None, version=None):
 register_view(RegistrationUserAccessViewSet, "registration_user_access")
 
 
+class SignUpListMixin:
```

Comment:
Instead of having these filters as these custom methods you could consider refactoring this logic into a `django_filters.rest_framework.FilterSet`

Reply:
I second this. We should move toward automatic documentation generation, and therefore having all new features support that is highly desirable.

Reply:
This one's done now - re-requesting reviews soon.